cmd/kubelet/app/server.go

```diff
@@ -216,7 +216,7 @@ func NewKubeletServer() *KubeletServer {
 		RootDirectory:                  defaultRootDir,
 		SerializeImagePulls:            true,
 		StreamingConnectionIdleTimeout: 5 * time.Minute,
-		SyncFrequency:                  10 * time.Second,
+		SyncFrequency:                  1 * time.Minute,
```

Comment:
The primary focus of this PR is to reduce redundant syncs (10 second periodic syncs) when there is no change. The added PLEG already checks the container runtime frequently for any container changes. We keep the periodic sync as a fallback, in case anything is missed, but it should be less frequent.